services/QuillLMS/app/models/school.rb

```diff
@@ -190,7 +190,7 @@ def generate_leap_csv(activities_since = Date.parse('2010-01-01'))
   end
 
   def students
-    User.joins(student_in_classroom: { teachers: :school }).where(schools: { id: id }).distinct
+    User.joins(student_in_classroom: { teachers: :school }).where(schools: { id: id }).where(classrooms_teachers: { role: ClassroomsTeacher::ROLE_TYPES[:owner] }).distinct
```

Comment:
This has gotten long enough that it might be worth breaking up over multiple lines the way we do in `vitally_school_stats.rb`